dist/flatpak/com.github.input-leap.input-leaps.yml

```diff
@@ -9,35 +9,58 @@
 # Note: mDNS (foo.local) does NOT work, so server.name needs to be a FQDN or an ip address
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
-app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+app-id: io.github.input-leap.input-leap
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: input-leap
 
 finish-args:
     # We want to talk to the network
     - --share=network
     # Need to read our config from somewhere
     - --filesystem=xdg-config/input-leap
-    # HACK: so we can put our EIS socket somewhere accessible
-    # This won't be needed once the portal actually supports our interfaces
-    - --filesystem=xdg-run/eis-0
-    # Let's show off :)
-    - --nosocket=x11
-    - --nosocket=wayland
+    - --socket=x11
+    - --socket=wayland
+    - --device=dri           # For hardware acceleration if needed
```

Comment:
Since we are not doing any 3d or video processing i don't think we need to have `dri`